src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.ParameterMatching.cs

```diff
@@ -500,6 +500,9 @@ public async Task TupleDeserializationWorks()
 
         [Fact]
         [DynamicDependency(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.PublicProperties, typeof(Tuple<,,,,,,,>))]
+#if !BUILDING_SOURCE_GENERATOR_TESTS
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/73347", typeof(PlatformDetection), nameof(PlatformDetection.IsNotBuiltWithAggressiveTrimming))]
```

Comment:
It should be ConditionalFact instead.

This should not be ActiveIssue. There is no better way to fix this.

Reply:
Given that we're already specifying `DynamicallyAccessedMemberTypes.PublicConstructors` shouldn't the parameter names not be trimmed by the linker? While they're technically not required by the constructor to function it rubs me as slightly aggressive.

Reply:
I just saw your comment about DynamicDependency dependency in the issue and replied: https://github.com/dotnet/runtime/issues/73347#issuecomment-1205293266